charts/linkerd2/templates/proxy-injector-rbac.yaml

```diff
@@ -76,6 +76,8 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   name: linkerd-proxy-injector-webhook-config
+  annotations:
+    {{- toYaml .Values.proxyInjector.annotations | nindent 4 }}
```

Comment:
A check around this block will be required, to avoid rendering an empty `annotations:` entry